### PR TITLE
fix(platform): fix a11y issue for object status component

### PR DESCRIPTION
**Pull request by manu-kr**
#### Please provide a link to the associated issue.
Closes https://github.com/SAP/fundamental-ngx/issues/5768
#### Please provide a brief summary of this pull request.
This PR contains changes to fix a11y issues for  platform object status component.  

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application

Documentation checklist:
- [NA] update `README.md`
- [NA] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

File: apps/docs/src/app/platform/component-docs/platform-object-status/platform-object-status-example/platform-object-status-clickable-and-icon-example.component.html [hunk 1]
```diff
@@ -42,8 +47,9 @@
     <fdp-object-status
         [indicationColor]="item"
         [label]="'Indication Color' + item"
+        [title]="'Indication Color' + item"
         clickable="true"
         (objectStatusClick)="showAlert(item)"
-        ariaLabel="Object status indication Color Clickable"
+        [ariaLabel]="'Object status Indication Color' + item + 'Clickable'"
```
[the remark this answers is not in the document]
Comment (kavya-b) [reply apps/docs/src/app/platform/component-docs/platform-object-status/platform-object-status-example/platform-object-status-clickable-and-icon-example.component.html:53]: [As per this comment](https://github.com/SAP/fundamental-ngx/pull/6297#discussion_r696312790) in another PR, dismissing the above comment as we are not sure what is the right way to deal with translations of default values to properties. Seems it was agreed for now that it is on the end user to provide the correct aria-label.

File: apps/docs/src/app/platform/component-docs/platform-object-status/platform-object-status-example/platform-object-status-clickable-and-icon-example.component.ts [hunk 2]
```diff
@@ -7,7 +7,7 @@ import { Component } from '@angular/core';
 })
 export class PlatformObjectStatusClickableAndIconExampleComponent {
     showAlert(index: number): void {
-        alert('you clicked the clickable ObjectStatus' + index);
+        alert('you clicked the clickable ObjectStatus       ' + index);
```
[the remark this answers is not in the document]
Comment (manu-kr) [reply apps/docs/src/app/platform/component-docs/platform-object-status/platform-object-status-example/platform-object-status-clickable-and-icon-example.component.ts]: Reverted changes.

File: apps/docs/src/app/platform/component-docs/platform-object-status/platform-object-status-example/platform-object-status-generic-text-example.component.html [hunk 3]
```diff
@@ -2,6 +2,7 @@
     <fdp-object-status
         [indicationColor]="item"
         [label]="'Indication Color'+item"
-        ariaLabel="Object status Indication color"
+        [title]="'Indication Color'+item"
+        [ariaLabel]="'Object status Indication color'+item"
```
[the remark this answers is not in the document]
Comment (manu-kr) [reply apps/docs/src/app/platform/component-docs/platform-object-status/platform-object-status-example/platform-object-status-generic-text-example.component.html]: Formatted.

File: libs/platform/src/lib/components/object-status/object-status.component.html [hunk 4]
```diff
@@ -1,5 +1,6 @@
 <span
     fd-object-status
+    [attr.role]="clickable? 'button' : null"
```
Comment (kavya-b) [inline libs/platform/src/lib/components/object-status/object-status.component.html:3]: formatting here

**Review by kavya-b — APPROVED** [review]
LGTM

File: libs/platform/src/lib/components/object-status/object-status.component.html (hunk 4 above)
Comment (manu-kr) [reply libs/platform/src/lib/components/object-status/object-status.component.html]: Done.

**Review by Smiranin — APPROVED** [review]

**Review by platon-rov — APPROVED** [review]

**Review by InnaAtanasova — COMMENTED** [review]
Given that the title value is the same as the label value, doesn't it make sense to set it automatically behind the scene and save the user the effort to provide both? For example, if the user does not provide value for `title`, it would be set to the value provided for `label`. If the user provides a value, it will use this value.

**Comment by puru-hk** [comment]
All issues are fixed.

**Review by puru-hk — APPROVED** [review]

**Review by Betrozov — APPROVED** [review]

**Comment by manu-kr** [comment]
> Given that the title value is the same as the label value, doesn't it make sense to set it automatically behind the scene and save the user the effort to provide both? For example, if the user does not provide value for `title`, it would be set to the value provided for `label`. If the user provides a value, it will use this value.

@InnaAtanasova Addressed this comment.

**Review by mikerodonnell89 — APPROVED** [review]

**Review by InnaAtanasova — APPROVED** [review]

**Pull request merged**